package.json

```diff
@@ -36,13 +36,15 @@
     "@typescript-eslint/eslint-plugin": "^2.3.0",
     "@typescript-eslint/parser": "^2.3.0",
     "@yarnpkg/pnpify": "^2.0.0-rc.20",
-    "cypress": "9.2.0",
```

Comment:
Why is cypress removed as a dependency?

Reply:
@logan-anderson for some reason having this in the root package.json was causing the my ide to infer that mocha was the test runner for the @tinacms/graphql and @tinacms/datalayer projects.

I'll put it back before this is taken out of draft mode and I have time to figure out why its screwing up the test runner